### PR TITLE
[RELENG-1091] migrate to action

### PR DESCRIPTION
**Pull request by erwan-b**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - name: Install node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '10'
+          cache: 'yarn'
+      - run: |
+          sudo apt-get update -q && sudo apt-get -qy install curl apt-transport-https
+          sudo apt-get install -qy software-properties-common
+          sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+          sudo apt-get update -q
```
Comment (tcarmet) [inline .github/workflows/test.yaml]: Have you tested without this?

**Review by tcarmet — COMMENTED** [review]
Can you please read the RELEASE.md doc and adapt the content of it for how release are made now?

**Review by anurag4DSB — APPROVED** [review]

**Review by mrepop — APPROVED** [review]

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildk
```
Comment (miniscruff) [inline .github/workflows/release.yml]:
```suggestion
      - name: Set up Docker Buildx
```

**Review by tcarmet — APPROVED** [review]

**Review by anurag4DSB — APPROVED** [review]

**Review by miniscruff — APPROVED** [review]

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to be released'
+        required: true
+      prerelease:
+        description: "Is the release a GA? (true / false)"
```
Comment (jonathan-gramain) [inline .github/workflows/release.yml]: The description "is it a GA?" seems to be the opposite of the section name "is it a prerelease?", which one is right?

**Review by jonathan-gramain — APPROVED** [review]

**Comment by erwan-b** [comment]
ping

**Comment by erwan-b** [comment]
ping

**Pull request merged**